docker-compose-sqlite.yaml

```diff
@@ -1,6 +1,26 @@
 version: '3.8'
 
 services:
+  gateway:
+    build:
+      context: .
+      dockerfile: gateway/docker/Dockerfile
+    # Note that, golem_gateway_port should be changed carefully
+    # as it is currently set to match the default port in golem-cli:
+    # Golem base url. Default: GOLEM_BASE_URL environment variable or http://localhost:9881
+    ports:
+      - "${GOLEM_GATEWAY_PORT}:80"
+    environment:
+      - GOLEM_WORKER_SERVICE_HOST=golem-worker-service
+      - GOLEM_WORKER_SERVICE_PORT=${WORKER_SERVICE_HTTP_PORT}
+      - GOLEM_TEMPLATE_SERVICE_HOST=golem-template-service
+      - GOLEM_TEMPLATE_SERVICE_PORT=${TEMPLATE_SERVICE_HTTP_PORT}
+    command: /bin/bash -c "envsubst < /etc/nginx/nginx.conf.template > /etc/nginx/nginx.conf && nginx -g 'daemon off;'"
```

Comment:
if we have custom docker image i think we should add this to startup script, so just configuration will be here

Reply:
The env variables are valid only at docker-compose stage. I don't think command for env substitution be used unless we set the env within Dockerfile which is a bad practice, especially it defaults to empty strings